### DISCUSSION TITLE
Raspberry Pi 64-bit?

### DISCUSSION THREAD

**Discussion by frankmilliron** [Orama-Interactive/Pixelorama#796] [Ideas]
Hello.  I tried installing Pixelorama from Snapcraft Store and it didn't launch. I'm on a 64-bit version of Raspbian, so I'm guessing the app is only compiled for the (admittedly more-common) 32-bit version of Raspbian. Would like to see a build for Raspberry Pi 64!
Cool app. My kids love making animations with it!

**Comment by OverloadedOrama** [comment]
Closing as duplicate of #594. 64-bit ARM builds will be available for Pixelorama v1.0.
